docs/Course Wiki/ECE Course Offerings/ECE498SJP.md

```diff
@@ -0,0 +1,59 @@
+---
+tags:
+- Hardware
+- Computer architecture
+- Digital design
+---
+# ECE498SJP
+
+ECE 498 SJP (Accelerator Architectures) is a 4-credit-hour course that can currently only be used to fulfill technical elective credit, should it be offered again in the near future.
```

Comment:
Change to "is a 4-credit-hour course that satisfies the Technical Electives requirement for ECE majors."